pkg/util/steps/runner.go

```diff
@@ -18,7 +18,12 @@ import (
 
 // FriendlyName returns a "friendly" stringified name of the given func.
 func FriendlyName(f interface{}) string {
-	return runtime.FuncForPC(reflect.ValueOf(f).Pointer()).Name()
+	return strings.TrimSuffix(runtime.FuncForPC(reflect.ValueOf(f).Pointer()).Name(), "-fm")
+}
+
+// FriendlyName returns a "friendly" stringified name of the given func.
+func ShortFriendlyName(f interface{}) string {
+	return strings.TrimPrefix(FriendlyName(f), "github.com/Azure/ARO-RP/")
```

Comment:
If [this](https://github.com/Azure/ARO-RP/pull/3422/files#diff-4013a8e6a28b1efd731c440735d77847e53bb080cf7d1b9a79abbc2beb80d668R110) is the only place we are still using the old FriendlyName() function, should we just incorporate your changes directly to `FriendlyName()` rather than introduce this new function, and just adjust the logic in `enrichConditionTimeoutError()` if needed?

Reply:
I agree with Caden here, plus the request on the shortName change and this will really clean it up.